### ISSUE TITLE
Need apache config for HTTP -- HTTPS working, HTTP not

### ISSUE THREAD

**Issue by coamtaker618** [RocketChat/Rocket.Chat#18526]
TL;DR:

 I have a working HTTPS apache config but I do not have a working HTTP apache config (IKR?).  For reference, my https config file is almost identical to the example provided here (https://docs.rocket.chat/installation/manual-installation/configuring-ssl-reverse-proxy) for reverse proxying of a domain.

Full Story:

This may well be the weirdest (read: dumbest) problem I've come across, but I have a working RocketChat on HTTPS and have been using it for months. Specifically 3 months; which I know as that is the length of a Lets Encrypt cert.

This is no big deal, except my http config disappeared :(   So LE/certbot is unable to run, because it relies on http >_>

I have tried putting in a default config file & modifying it to my domain to no avail.  If anyone has an example that they could share that would be greatly appreciated :)

For reference, my https config file is almost identical to the example provided here (https://docs.rocket.chat/installation/manual-installation/configuring-ssl-reverse-proxy) for reverse proxying of a domain.
